chart/flux/templates/ssh.yaml

```diff
@@ -15,4 +15,14 @@ data:
         {{ .Values.ssh.known_hosts }}
       {{- end }}
     {{- end }}
+  config: |
+    {{- if .Values.ssh.config }}
```

Comment:
When a config is not specified, this creates an empty file. Can you please move the `if` to include the the config entry.

Reply:
That makes total sense. Pushed a new commit. Done a quick validation too. Thanks a ton reviewing.